index.html

```diff
@@ -1230,9 +1228,9 @@ <h3>Avoid correlation</h3>
       </p>
 
       <p>
-        A DID method implementer should rotate keys and identifiers as often as
-        possible
-        to avoid correlation
+        DID method implementations are encouraged to rotate keys and
+        identifiers as often as possible,
+        to avoid correlation.
```

Comment:
I know that this was already in the implementer guide, but it makes no sense. If you have a DID that is static, it makes no sense to rotate keys often in order to avoid correlation.

Reply:
Also, "rotating keys" generally does not avoid correlation. Using different identifiers for different contexts does, however.

Reply:
Agree, key rotation is about mitigating compromise... not mitigating correlation....

But the part about identifiers is correct, generating a new short lived DID and using it for 1 purpose, and then rotating away from it without a network observing seeing that rotation does mitigate correlation.

did key comes to mind in this regard, I think this section can be improved.... very good catch and comments.

Reply:
Changed: https://github.com/w3c/did-imp-guide/pull/37#discussion_r762365123